server/src/main/java/org/candlepin/model/VirtConsumerMap.java

```diff
@@ -74,6 +74,10 @@ public Consumer get(String virtUuid) {
         return null;
     }
 
+    public java.util.Collection<Consumer> getConsumers() {
```

Comment:
A bit more PoC code that's left over. We should do a proper import and not fully qualify this on the fly.